guides/10_Video_Processor.md

```diff
@@ -34,26 +33,28 @@ WebRTC Unified Plan is also required. Unified Plan is by default enabled in [Mee
+`VideoTransformDevice` allows `VideoFrameProcessor`s to be applied to to a `Device` and provide a new object which can be passed into `meetingSession.audioVideo.startVideoInput`.
 
 `DefaultVideoTransformDevice` is the provided implementation of `VideoTransformDevice`. It requires four parameters: (1) `Logger` (2) `Device` (3) `Array<VideoFrameProcessor>`. 
 The `DefaultVideoTransformDevice` uses `VideoFrameProcessorPipeline` under the hood and hides its complexity.
 
 #### Construction and Starting Video Processing
 
-The construction of the `DefaultVideoTransformDevice` will not start the camera or start processing. The method `meetingSession.audioVideo.chooseVideoInputDevice` is needed to be called. The device controller will use the inner `Device` to acquire the source `MediaStream` and start the processing pipeline at the same frame rate.
+The construction of the `DefaultVideoTransformDevice` will not start the camera or start processing. The method `meetingSession.audioVideo.startVideoInput` is needed to be called. The device controller will use the inner `Device` to acquire the source `MediaStream` and start the processing pipeline at the same frame rate.
 "Inner device" in this context refers to the original video stream coming from the selected camera.
 The parameters to `chooseVideoInputQuality` are used as constraints on the source `MediaStream`. 
 After the video input is chosen, `meetingSession.audioVideo.startLocalVideoTile` can be called to start streaming video.
 
 #### Switching the Inner Device on VideoTransformDevice
 
 To switch the inner `Device` on `DefaultVideoTransformDevice`, call `DefaultVideoTransformDevice.chooseNewInnerDevice` with a new `Device`.
-`DefaultVideoTransformDevice.chooseNewInnerDevice` returns a new `DefaultVideoTransformDevice` but preserves the state of `VideoFrameProcessor`s. Then call `meetingSession.audioVideo.chooseVideoInputDevice` with the new transform device. 
+`DefaultVideoTransformDevice.chooseNewInnerDevice` returns a new `DefaultVideoTransformDevice` but preserves the state of `VideoFrameProcessor`s. Then call `meetingSession.audioVideo.startVideoInput` with the new transform device. 
 
 #### Stopping VideoTransformDevice
 
-To stop video processing for the chosen `DefaultVideoTransformDevice`, call `meetingSession.audioVideo.chooseVideoInputDevice` with a different `DefaultVideoTransformDevice` or `null` to stop using previous `DefaultVideoTransformDevice`.  The method `meetingSession.audioVideo.stopLocalVideoTile` can also be used to stop the streaming.
+To stop video processing for the chosen `DefaultVideoTransformDevice`, call `meetingSession.audioVideo.
+startVideoInput` with a different `DefaultVideoTransformDevice` or call `meetingSession.audioVideo.stopVideoInput` to 
+stop using previous `DefaultVideoTransformDevice`.
```

Comment:
nit: remove the newline inside inline code

```
`meetingSession.audioVideo. <==== remove new line here
startVideoInput`
```
renders like this:
`meetingSession.audioVideo.
startVideoInput`